CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+# V2.4
+
+## Enhancements:
+
+- `starting-project` input added
+
+## Bugs:
+
+- Error treating export commands like `export DROOLS_VERSION=mvn org.apache.maven.plugins:maven-help-plugin:evaluate -Dexpression=project.version | grep ^7` will execute `export DROOLS_VERSION=mvn org.apache.maven.plugins:maven-help-plugin:evaluate -Dexpression=project.version | grep ^7 -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -B`. Fixed.
```

Comment:
What about adding a phrase like "It is now possible to export values using a command that starts as `export ...`"

Reply:
@danielezonca where do you mean? In README.md? In any case my POV is closer to "you can run whatever you want to execute" than to specify what  you can do. So in case you are not able to execute anything, the tool should fix this case.